### PR TITLE
[bitnami/mysql] Unify and document production values

### PR DESCRIPTION
**Pull request by carrodher**

### CODE REVIEW

File: bitnami/mysql/values.yaml [hunk 1]
```diff
@@ -28,7 +28,6 @@ image:
 
 service:
   ## Kubernetes service type
-  # nodePort:
```
Comment (carrodher) [inline bitnami/mysql/values.yaml]: Not sure about this one, it doesn't appear in the production one and it is not documented cc @tompizmor

Comment (tompizmor) [reply bitnami/mysql/values.yaml]: Is it used in the service? Maybe we forgot to document it..

Comment (carrodher) [reply bitnami/mysql/values.yaml]: No, it is not used:
```
$ ag 'nodePort' bitnami/mysql
bitnami/mysql/values.yaml
31:  # nodePort:
```

Comment (tompizmor) [reply bitnami/mysql/values.yaml]: then we should delete it from the values.yaml

**Pull request merged**